javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,68 @@
+// Flight booking fullname function
+
+const useFormalName = "Lord" + getFullname;
+
+getFullname("Benjamin", "Hughes", useFormalName);
+let fullname1  = "Benjamin";
+let fullname2  = "Hughes";
+function getFullname(fullname1, fullname2) {
+  if (useFormalName) {
+  return ("Lord" + " " + fullname1 + " " + fullname2)
+} else {
+  return (fullname1 + " " + fullname2)
+}
+}
+getFullname(fullname1, fullname2);
+/* the getFullname for women will work the similar way: we need to create boolean variable useWomenName and if it's true
+we should use Queen as a formal way instead of Lord.
+*/
+
+// Event application
+function getEventWeekday(){
+let days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]
+const todaysDate = new Date();
+
+}
+// Weather wear
+
+function whatToWear(a) {
+  if (a >= 0 && a <= 15) {
+    console.log("Wear jeans and a jacket")
+  } else if (a >= 16) {
+    console.log("Wear top and shirts")
+  } else {
+    console.log("Wear a space suit - it is too cold!")
+  }
+}
+const clothesToWear = whatToWear(5);
+console.log(clothesToWear);
+
+// Student manager
+
+const class07Students = [];
+const queenName = "Queen";
+function addStudentToClass(studentName) {
+for (let i = 0; i < 6; i++) {
```

Comment:
No need for this loop here. What this does is just add the `studentName` to the class, and then return the array, thus ending the function. This means that everything that is below this loop will never be run. And the student is always added, meaning this function can be called on the same student infinite times. Remove the loop, and start the function with the `if` statements.